bitnami/harbor/templates/exporter/exporter-dpl.yaml

```diff
@@ -99,14 +99,27 @@ spec:
               value: {{ ternary "true" "false" .Values.exporter.image.debug | quote }}
             - name: HARBOR_DATABASE_PASSWORD
               valueFrom:
+              valueFrom:
+                {{- if .Values.exporter.existingSecret }}
+                secretKeyRef:
+                  name: {{ .Values.exporter.existingSecret }}
```

Comment:
I don't see clearly the use case but I think we should use here something like [this](https://github.com/bitnami/charts/blob/443d63e96275bea2525ed0760917f35b443d6154/bitnami/harbor/templates/core/core-dpl.yaml#L145-L149) because the passwords you are looking for should be in `-envvars` secret

Reply:
The exporter pulls envvars from Harbor-core-envvars for the admin password and postgres password. For my use case i have set the postgres password in the chart as $POSTGRES_PASSWORD to avoid plain text password which then causes the exporter to think the password is literally "$POSTGRES_PASSWORD". I set extraenvvars secret but it does not over write the harbor-core-envvar. I hope this made sense. Thats why I was hoping the chart could do either default harbor-core-envvar or an existing secret option. Otherwise we could have externaldatabase.existingSecret for password and same for admin password. Please let me know if im missing something

Reply:
I just seen you had a link to core-envvars. Yeah that could work aswell. I think the exporter only pulls from the harbor-core-envvars instead of having an else statement which is causing it to not be overwritten by a different secret.

Reply:
Please see my comment below. 

About _setting the postgres password in the chart as `$POSTGRES_PASSWORD`_, I am not sure about how you are doing it, but if you want to use an environment variable already set in your PC or in the machine where you are running the deployment command you can set the helm chart value with the `--set` option in helm:
```console
helm install harbor oci://registry-1.docker.io/bitnamicharts/harbor --set externalDatabase.password=$POSTGRES_PASSWORD
```

Reply:
I am using ArgoCD to deploy harbor as an applicationset. I use ExternalSecrets or Vault for all sensitive data including PSQL passwords.